### ISSUE TITLE
[helm] add gateway similar to tempo/loki

### ISSUE THREAD

**Issue by rlex** [grafana/phlare#442]
#### Is your feature request related to a problem? Please describe.

Will be good if we can add gateway for ingress like in tempo/loki charts with http auth. Maybe port it directly from that charts?

**Comment by simonswine** [comment]
Yes fully agreed. We haven't gotten around to it yet, and it might be a couple more weeks until we get to it. 

If you feel up for it I am more than happy to review/accept/help PRs that do exactly what your suggesting.
